src/nvim/ex_docmd.c

```diff
@@ -5815,7 +5815,7 @@ char_u *get_user_cmd_complete(expand_T *xp, int idx)
   }
   char *cmd_compl = get_command_complete(idx);
   if (cmd_compl == NULL) {
-    return "\000";
```

Comment:
just NULL. Thus the `if` can be removed.

Reply:
No, in ExpandGeneric function, there is a check for NUL,

Reply:
as I investigated, NUL is '\000', so

Reply:
which is same as "", thus made the small change

Reply:
NUL and NULL are used differently

Reply:
you're right, it should work.